File: api/webAPI/WebApplication1/Models/Oprema.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace WebApplication1.Models
{
    public class Oprema
    {
        public int idOpreme { get; set; } // dodaje se u bazi
        public string Naziv { get; set; } // dodaje se u bazi
        public string Tip { get; set; }
        public string Kordinate { get; set; }
        public string Adresa { get; set; }

    }
}

File: angular-web2-app/src/app/entities/oprema/oprema.ts
export class Oprema {
    idOpreme:number;
    Naziv:string;
    Tip:string;
    Kordinate:string;
    Adresa:string;

    constructor(idOpreme:number, Naziv:string, Tip:string, Kordinate:string, Adresa:string) {
        this.idOpreme = idOpreme;
        this.Naziv = Naziv;
        this.Tip = Tip;
        this.Kordinate = Kordinate;
        this.Adresa = Adresa;
    }
}